<launch>
  <arg name="frame_name" default="world"/>
  <arg name="robot_pose_control_msg" default="/firefly/command/pose"/>

  <arg name="robot_pose_x" default="0.0"/>
  <arg name="robot_pose_y" default="0.0"/>
  <arg name="robot_pose_z" default="1.0"/>
  <arg name="robot_pose_yaw" default="0.0"/>

  <node name="publish_init_robot_pose_command" pkg="tentabot" type="publish_robot_pose_command" output="screen">
    <param name="frame_name"                  value="$(arg frame_name)"/>
    <param name="robot_pose_control_msg"      value="$(arg robot_pose_control_msg)"/>
    <param name="robot_pose_x"                value="$(arg robot_pose_x)"/>
    <param name="robot_pose_y"                value="$(arg robot_pose_y)"/>
    <param name="robot_pose_z"                value="$(arg robot_pose_z)"/>
    <param name="robot_pose_yaw"                value="$(arg robot_pose_yaw)"/>
  </node>
</launch>
